python_scripts/mongo_connection.py

```diff
@@ -0,0 +1,8 @@
+from pymongo import MongoClient
+
+
+def connect():
+    client = MongoClient('34.243.235.124', 27017)
```

Comment:
It's worth creating a `.env.example` file to specify which are required and loading these in via os.get.environ; share details via private slack channel for now;

```
import os
host = os.environ['host']
```